debug/genConnectionMetadata.js

```diff
@@ -1,53 +1,59 @@
 /* eslint-disable  @typescript-eslint/no-var-requires */
-const crypto = require('crypto');
+const { api } = require('@replit/protocol');
+const paseto = require('./paseto');
 
-const { TOKEN_SECRET } = process.env;
+const keyId = process.env.USER_KEY_ID;
+const govalPrivateKey = process.env.USER_PRIVATE_KEY_PEM.replace(/\\n/, '\n');
+const govalPublicKey = process.env.USER_PUBLIC_KEY_PEM.replace(/\\n/, '\n');
```

Comment:
```suggestion
const govalPrivateKey = crypto.createPrivateKey(process.env.USER_PRIVATE_KEY_PEM.replace(/\\n/, '\n'));
const govalPublicKey = crypto.createPublicKey(process.env.USER_PUBLIC_KEY_PEM.replace(/\\n/, '\n'));
```

the paseto implementation expects the keys to be parsed and converted to node objects prior to using them.

Reply:
Right, thanks for suggestion